web/sites/default/default.settings.php

```diff
@@ -738,7 +738,7 @@
  * well as the original entity type and field storage definitions should be
  * retained after a successful entity update process.
  */
-$settings['entity_update_backup'] = TRUE;
+$settings['entity_update_backup'] = FALSE;
```

Comment:
@RichardDavies , this would need to be added to `web/sites/default/settings.php` or `web/sites/default/settings.pantheon.php` to have any impact. `web/sites/default/default.settings.php` is a template for settings.php; it does not load.